City-of-Helsinki/drupal-hdbt
# UHF-8944 CKEditor5 blockquote element structure

# [UHF-8944](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8944)


## What was done


* Added data-attributes to blockquote styles.

## How to install and test
* Check install and test instructions from https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/599

## Designers review



## Other PRs


* https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/599



[UHF-8944]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8944?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
